clients/lsp-html.el

```diff
@@ -182,9 +206,13 @@ styles."
                                      (cons (lsp-package-path 'html-language-server)
                                            lsp-html-server-command-args)))
                   :activation-fn (lsp-activate-on "html")
+                  :major-modes '(html-mode)
```

Comment:
Why do we need this since we already have `:activation-fn`. 🤔

Reply:
Had some issues with lsp-mode not starting for html files and giving me an error message on no servers available. Removed the line and will investigate the issues further. Might be some weirdness with my local setup.


Removed the major-mode line and rebased 🙂